fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/LandPathNodeTypesRegistry.java

```diff
@@ -55,7 +56,7 @@ private LandPathNodeTypesRegistry() {
 	 *                           (Pass {@code null} to not specify a node type and use the default behavior)
 	 */
 	public static void register(Block block, @Nullable PathNodeType nodeType, @Nullable PathNodeType nodeTypeIfNeighbor) {
-		Objects.requireNonNull(block, "Block cannot be null!");
+		Preconditions.checkNotNull(block, "Block cannot be null!");
```

Comment:
I would prefer `Objects.requireNonNull`. We avoid guava if possible.

Reply:
They are the same implementation, also I prefer Objects because is java native.
...however in fapi i found some modules with Preconditions and some with Objects. 🤔

Reply:
The Objects method is used a lot more in Fabric API so it's also better for consistency (the Preconditions null check callers are outliers imo).